adding drupal blog post (#5814)
unlock-protocol/unlock: unlock-protocol.com/blog/drupal-plugin.md

---
title: Drupal Plugin
subTitle: Easily add lock to a Drupal site!
authorName: Julien Genestoux
publishDate: January 24, 2020
description: Drupal is one of the most popular Content Management Systems. You can now add locks to a Drupal site easily!
image: /static/images/blog/drupal/unlock-drupal.jpg
---

Content Management Systems make it easy for people to create their own websites. [Drupal](https://www.drupal.org/) is one of the most popular CMS. Drupal is open source and developers can easily customize each Drupal instance. Today, they're also able to easily add locks to limit access to content of features to visitors which own keys to these locks!

> A couple of weeks ago, external contributor [Mike Purvis](https://twitter.com/mikedotexe) asked if Unlock would work with Drupal. We quickly offered him to work on the plugin and you can now [install it on your own Drupal website](https://www.drupal.org/project/unlock)! It is also [fully open source too](https://github.com/mikedotexe/unlock), so you can of course help improve it!

Reviewer: ```suggestion
A couple of weeks ago, external contributor [Mike Purvis](https://twitter.com/mikedotexe) asked if Unlock would work with Drupal. He quickly rose to the challenge and worked on a plugin; you can now [install it on your own Drupal website](https://www.drupal.org/project/unlock)! It is also [fully open source too](https://github.com/mikedotexe/unlock), so you can of course help improve it!
```
